tests/test_latent_infection_process.py

```diff
@@ -48,7 +48,7 @@ def test_LatentInfectionProcess():
     )
 
     with numpyro.handlers.seed(rng_seed=223):
-        latent_inf_w_hierarchical_effects = my_latent_infection_model(
+        latent_inf_w_hierarchical_effects, *_ = my_latent_infection_model(
```

Comment:
There's only a single other output, so use a single underscore to throw that output away. That way, if there are unexpected additional outputs, the code will error.
```suggestion
        latent_inf_w_hierarchical_effects, _ = my_latent_infection_model(
```

Reply:
done in https://github.com/CDCgov/pyrenew-hew/pull/310/commits/783425aaf46edaf5c27fe3ddfcd7bd53e4d7208a